src/github.js

```diff
@@ -964,6 +964,18 @@
                body: comment
             }, cb);
          };
+         this.edit = function(issue,options,cb){
```

Comment:
The snippet doesn't follow our code convention style. Can you use our `lint` task so that it will automatically fix it for you?